tests/quadsim/robots/test_motors.py

```diff
@@ -8,23 +8,9 @@
 from quadsim.robots.motors import MotorModel
 
 
-@pytest.mark.parametrize(
-    "motor_command, expected_applied_torque, expected_observed_torque",
-    [
-        pytest.param((20.0, 30.0), (20.0, 30.0), (20.0, 30.0), id="0",),
-        pytest.param((50.0, 30.0), (40.0, 30.0), (50.0, 30.0), id="1",),
-        pytest.param((-20.0, 60.0), (-20.0, 50.0), (-20.0, 60.0), id="2",),
-        pytest.param((-70.0, -60.0), (-60.0, -50.0), (-70.0, -60.0), id="3",),
-    ],
-)
-def test_torque_control(
-    motor_command: Tuple[float, float],
-    expected_applied_torque: Tuple[float, float],
-    expected_observed_torque: Tuple[float, float],
-) -> None:
-
-    # instantiate MotorGroup (containing 2 MotorModels)
-    motors = MotorGroup(
+@pytest.fixture
```

Comment:
Nice. This is the technically sound way of doing this in pytest. I didn't catch this earlier!